packages/gatsby-theme-docs/src/templates/page-content.js

```diff
@@ -106,7 +54,11 @@ export default PageContentTemplate;
 
 export const query = graphql`
   query QueryMarkdownPage($slug: String!) {
-    mdx(fields: { slug: { eq: $slug } }) {
+    contentPage(slug: { eq: $slug }) {
+      title
+      beta
+      isGlobalBeta
+      excludeFromSearchIndex
```

Comment:
Those fields should be queried here, we don't need to pass them with the page context.
Also note the we are using the `contentPage` query field.